docs/user/config-file/v2.rst

```diff
@@ -473,7 +473,7 @@ Specify a list of commands that Read the Docs will run on the build process.
 When ``build.commands`` is used, none of the :term:`pre-defined build jobs` will be executed.
 (see :doc:`/build-customization` for more details).
 This allows you to run custom commands and control the build process completely.
-The ``output/`` directory (relative to the checkout's path) will be uploaded and hosted by Read the Docs.
+The ``_readthedocs/html`` directory (relative to the checkout's path) will be uploaded and hosted by Read the Docs.
```

Comment:
This still feels really small compared to how important the information is, but we can improve it over time.